src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -733,14 +752,24 @@ private void refreshBeansList() throws IOException {
                     ObjectName name = new ObjectName(scope);
                     this.beans.addAll(connection.queryNames(name));
                 }
-            } catch (Exception e) {
+                if (instanceTelemetryBean != null) {
+                    instanceTelemetryBean.setDomainsQueried(beanScopes.size());
+                }
+            } catch (MalformedObjectNameException e) {
```

Comment:
`setDomainsQueried` can throw a `MalformedObjectNameException`? 

Also since you catch both the exception types, I don't think think they can escape this method, so you should be able to remove the `throws` decl from the method signature I believ.

Reply:
`new ObjectName` can throw `MalformedObjectNameException`. removed the `throws` decl

Reply:
L752
https://github.com/DataDog/jmxfetch/blob/b9671c319a24bad27e4e6b07b30fd7d8c73ba886/src/main/java/org/datadog/jmxfetch/Instance.java#L749-L752